server/routes/creatingLicences/types/time.ts

```diff
@@ -0,0 +1,14 @@
+import { Expose } from 'class-transformer'
+
+class SimpleTime {
+  @Expose()
+  hour: string
+
+  @Expose()
+  minute: string
+
+  @Expose()
+  ampm: string
```

Comment:
Could this need an enumerated type for validation i.e. 'am' or 'pm'?

Reply:
e.g. @IsIn(['am,'pm'])

Reply:
Added this Enum, I still kept the validation of the enum together with the validation of the rest of the SimpleTime class though